spec/controllers/games_controller_spec.rb

```diff
@@ -136,4 +136,184 @@
       expect(response).to redirect_to(games_url)
     end
   end
+
+  # Adds white pieces to board
+
+  describe 'populate board' do
+    it 'Adds correct number of white Pawns to the board' do
+      game = Game.create!
+      game.populate_board!
+
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 0)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 1)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 2)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 3)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 4)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 5)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 6)
+      expect(game.pieces.where(type: 'Pawn', color: 'White')).to eq game.pieces.where(position_x: 6, position_y: 7)
+    end
+  end
+
```

Comment:
@mscoutermarsh Hey Mike, could you take a look at the code on this branch/PR instead of the other one? I rebased this from `master` last night, after merging the `render-pieces` branch, and also made a bunch of edits due to Travis offenses.